frontend/src/components/module-editor/EditableContentBlock.tsx

```diff
@@ -55,6 +57,12 @@ const EditableContentBlock = ({
               </svg>
             </Box>
             {SelectContentBlock(block)}
+            <EditContentOptionsMenu
+              isVisible={isHovered}
```

Comment:
Rather than passing in a prop, is there any reason not to just conditionally render this? i.e.
```suggestion
            {isHovered && <EditContentOptionsMenu
            ...
```